jenkins/jenkins.groovy

```diff
@@ -0,0 +1,52 @@
+//-------------------------------------------------------------------
+// File: jenkins.groovy
+// Description: A util file for hnsesb jenkins jobs
+//-------------------------------------------------------------------
+//This method returns namespace name with license plate prefix
+def project(val) {
+	return "c5839f-" + val.toLowerCase()
```

Comment:
Just confirming hardcoding this is OK as it's the prefix for our projects and won't change while we are on this platform.

Reply:
yes, this is ouor namespace's license plate (as they call it). It will not change for us till we are on silver cluster. If we switch to gold cluster, we will get a new license plate. With this method added in common util, updating it here will be effective for all jenkins jobs.